src/test/java/org/assertj/core/internal/files/Files_assertCanRead_Test.java

```diff
@@ -46,18 +42,19 @@ void should_fail_if_actual_is_null() {
 
   @Test
   void should_fail_if_can_not_read() {
-    when(actual.canRead()).thenReturn(false);
+    // GIVEN
+    File notAFile = new File("xyz");
```

Comment:
minor: rename to `nonExistentFile` which is more accurate as `notAFile` is a `File` but simply not an existing one